VotingApplication/VotingApplication.Web/App_Start/BundleConfig.cs

```diff
@@ -43,7 +43,7 @@ public static void RegisterBundles(BundleCollection bundles)
             ngDialogThemeCss.Include("~/Content/Lib/Css/ngDialog-theme-default-min.css");
             bundles.Add(ngDialogThemeCss);
 
-            StyleBundle mdDateTimeCss = new StyleBundle("~/Bundles/StyleLib/mdDateTime", "https://cdn.rawgit.com/SimeonC/md-date-time/v0.0.14/dist/md-date-time.css");
+            StyleBundle mdDateTimeCss = new StyleBundle("~/Bundles/StyleLib/AngularDateTime", "https://cdn.rawgit.com/SimeonC/md-date-time/v0.0.14/dist/md-date-time.css");
```

Comment:
Why has this changed? All the existing bundles that are prefixed with "Angular" are actually angular modules. Changing it to "AngularDateTime" implies that this is an angular module, from the google cdn, rather than a 3rd-party one from github.